src/main/java/no/finn/unleash/metric/ClientRegistration.java

```diff
@@ -44,8 +50,8 @@ public LocalDateTime getStarted() {
         return started;
     }
 
-    public long getInterval() {
```

Comment:
I needs to serialize as "interval" as this is the contract on the API side:
https://github.com/Unleash/unleash/blob/master/lib/services/client-metrics/register-schema.js#L17

Reply:
ah, good point.

Reply:
It should now be serialized as interval still.